src/app_service/service/profile/service.nim

```diff
@@ -193,6 +193,7 @@ QtObject:
       if rpcResponseObj{"error"}.kind != JNull and rpcResponseObj{"error"}.getStr != "":
         error "Error saving profile showcase preferences", msg = rpcResponseObj{"error"}
         return
+      self.requestProfileShowcasePreferences()
```

Comment:
NP: I originally queried from the database after saving, but that caused an extra animation